protobuf/src/buf_read_iter.rs

```diff
@@ -242,7 +242,7 @@ impl<'ignore> BufReadIter<'ignore> {
         }
     }
 
-    fn _read(&mut self, buf: &mut [u8]) -> ProtobufResult<usize> {
+    pub(crate) fn _read(&mut self, buf: &mut [u8]) -> ProtobufResult<usize> {
```

Comment:
`pub` is enough, since `BufReadIter` is not an exported type.

Could you rename function to `read`, without underscore please?